applications/FluidDynamicsApplication/python_scripts/navier_stokes_two_fluids_solver.py

```diff
@@ -106,6 +106,19 @@ def __init__(self, model, custom_settings):
                 custom_settings["levelset_convection_settings"].RemoveValue("levelset_splitting")
                 KratosMultiphysics.Logger.PrintWarning("NavierStokesTwoFluidsSolver", "\'levelset_splitting\' has been temporarily deactivated. Using the standard levelset convection with no splitting.")
 
+        #TODO: Remove this after the retrocompatibility period
+        if custom_settings.Has("bfecc_convection"):
+            KratosMultiphysics.Logger.PrintWarning("NavierStokesTwoFluidsSolver", "\'bfecc_convection\' is no longer supported. Using the standard levelset convection.")
+            custom_settings.RemoveValue("bfecc_convection")
+            if custom_settings.Has("bfecc_number_substeps"):
+                custom_settings.RemoveValue("bfecc_number_substeps")
+
+        #TODO: Remove this after the retrocompatibility period
+        if custom_settings.Has("algebraic_stabilization"):
+            KratosMultiphysics.Logger.PrintWarning("NavierStokesTwoFluidsSolver", "Using deprecated convection element mechanism with \'algebraic_stabilization\'. Use \'element_type\' within \'levelset_convection_settings\' instead. Default convection will be used")
```

Comment:
I think it has no user at the moment and you can omit this warning.

Reply:
Fine. But maybe we can keep it for a while (just in case you run one of your old cases).

Reply:
For me it is completely fine if you remove it now.